api/v1/relay/relay.proto

```diff
@@ -0,0 +1,48 @@
+// Copyright 2020 Authors of Cilium
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+
+package relay;
+
+// NodeStatusEvent is a message sent by hubble-relay to inform clients about
+// the state of a particular node.
+message NodeStatusEvent {
+    // state_change contains the new node state
+    NodeState state_change = 1;
+    // node_name is the list of nodes for which the above state changes applies
+    repeated string node_name = 2;
```

Comment:
I think it should be pluralized. This already confused me while reviewing https://github.com/cilium/hubble/pull/260 as I thought that `NodeName` was just a node name when in fact it's a list of node names.